src/arvcamera.c

```diff
@@ -1271,6 +1259,66 @@ arv_camera_get_frame_rate_bounds (ArvCamera *camera, double *min, double *max, G
 	}
 }
 
+/*
+* arv_camera_set_frame_rate_enabled:
```

Comment:
The feature name in Genicam SFNC is AcquisitionFrameRateEnable. Please rename the function to arv_camera_set_frame_rate_enable.